src/Util/TestListenerTrait.php

```diff
@@ -123,6 +123,10 @@ function {$f['name']}{$f['signature']}
                     $polyfillSignature = ReflectionCaster::castFunctionAbstract(new \ReflectionFunction($testNamespace.'\\'.$f['name']), [], new Stub(), true);
                     $polyfillSignature = ReflectionCaster::getSignature($polyfillSignature);
 
+                    if ('mb_get_info' === $r->name && false === strpos($originalSignature, '|null') && false !== strpos($polyfillSignature, '|null')) {
```

Comment:
can you add a comment about which PHP versions are concerned by this signature patch, to allow cleaning it in the future ?

Reply:
comment added